armi/physics/fuelCycle/fuelHandlerInterface.py

```diff
@@ -65,13 +65,15 @@ def interactBOC(self, cycle=None):
         # if lattice physics is requested, compute it here instead of after fuel management.
         # This enables XS to exist for branch searching, etc.
         mc2 = self.o.getInterface(function="latticePhysics")
+        xsgm = self.o.getInterface("xsGroups")
```

Comment:
Sorry, are we SURE this won't cause the xsGroups interface to run twice for anyone downstream?

Reply:
There is one case where it will run twice, but that is by design. This was done to support the original branch searching functionality [as described here](https://github.com/terrapower/armi/issues/839#issuecomment-1215163659)). 

@HunterPSmith @onufer